### PR TITLE
Document NodeLocalDNS forwardToKubeDNS breaking change

### LINKED ISSUE #13442: 1.22.0 Release Required actions missing NodeLocalDNS BC
/kind bug

**3. What cloud provider are you using?**

Cluster upgrade from 1.21 to 1.22.x

**4. What commands did you run?  What is the simplest way to reproduce this issue?**

cluster upgrade

**5. What happened after the commands executed?**

NodeLocalDNS stops forwarding to kubedns

**6. What did you expect to happen?**

Keep same behaviour as in 1.21

**7. Please provide your cluster manifest. Execute

```yaml
  kubeDNS:
    provider: CoreDNS
    coreDNSImage: k8s.gcr.io/coredns/coredns:v1.8.4
    nodeLocalDNS:
      enabled: true
```

`forwardToKubeDNS` default was behaviour was acting as `true`

**9. Anything else do we need to know?**

Add a note to `Required actions` section on the release of 1.22.0 for the next one to be aware of that important change on the behaviour.

### PR DESCRIPTION
**Pull request by jorge07**
Fixes #13442

### CODE REVIEW

**Review by olemarkus — APPROVED** [review]

**Comment by olemarkus** [comment]
/ok-to-test

Can you fix the commit message?

File: docs/releases/1.22-NOTES.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Currently this is only available using the AWS cloud provider.
 
 kOps can now provision instances with nvidia GPUs and configure it for container workloads without the need of hooks and operators. See [GPU support](https://kops.sigs.k8s.io/gpu/)
 
+## Breacking change in NodeLocalDNS
+
+Since 1.22.0 Cluster `spec.kubeDNS.nodeLocalDNS.forwardToKubeDNS` default behaviour changes from `true` to `false`.
```
Comment (ederst) [inline docs/releases/1.22-NOTES.md:39]: TIL, it probably would have been smarter to change the default behavior from 'false' to 'true' when fixing the template (https://github.com/kubernetes/kops/pull/11743).

Comment (jorge07) [reply docs/releases/1.22-NOTES.md:39]: That will generate another BC change. At current stage, documentation is enough IMO.

Comment (jorge07) [reply docs/releases/1.22-NOTES.md:39]: I think for the majority of people, false is the way to go. I'm ok with the false as default. Just need to be documented in the release.

Comment (olemarkus) [reply docs/releases/1.22-NOTES.md:39]: False is definitely the way to go. I think very few do custom core files.

**Comment by jorge07** [comment]
@olemarkus tell if that's correct now

**Review by olemarkus — APPROVED** [review]

**Pull request merged**